topology/m4/dai.m4

```diff
@@ -188,4 +188,14 @@ define(`DAI_ADD',
 `include($1)'
 )
 
+dnl VFE_LINK(link_name, cpu_dai_name, platform_name)
```

Comment:
Why do we need this ? it looks like we are defining a BE.

Reply:
@lgirdwood I cant get the dpcm_runtime_update() function to startup the codec without specifying the CPU dai name in the virtual FE. In the case of the up2 with hifiberry , this is "SSP5 Pin". So I thought the topology would be the best place to define it. Is there any other way I can get the cpu dai name while loading the siggen widget?

Reply:
ok, the tone should be routed using DAPM to the DAI or any other endpoint. So DAPM alone should route tone to SSP5. The VFE should have no string or other references (unless through DAPM) to any BE or BEs.  You may need to debug why the VFE needs a BE string reference if this does not work alone.